### DISCUSSION TITLE
How to use with Meta StyleX

### DISCUSSION THREAD

**Discussion by Tech-Dex** [radix-ui/themes#209] [Q&A]
Hello,

I've started a discussion on StyleX github as well around the question on "How to use with a component library (https://github.com/facebook/stylex/discussions/224)" .

They gave a great response, but it is a bit hacky as they state. I was wondering if Radix UIi plans to allow an easy use of StyleX without the hacky way? I am interested on how to achieve the conditional styling.

**Comment by vladmoroz** [comment]
Assuming you are asking about Radix Primitives, I replied in https://github.com/facebook/stylex/discussions/224#discussioncomment-7960888

Does this clear it up?
